cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/meta/RemoteNamespaceQueryHandler.java

```diff
@@ -35,24 +35,24 @@
 @Path(AbstractRemoteSystemOpsHandler.VERSION + "/execute")
 public class RemoteNamespaceQueryHandler extends AbstractRemoteSystemOpsHandler {
```

Comment:
Yea, the naming is not good.
I think we should remove this handler now.
@bdmogal @gokulavasan ?